### PR TITLE
Render rockets to rockets page

### LINKED ISSUE #18: [4pt] Display rockets - Lists render
- Use `useSelector()` Redux Hook to select the state slices and render lists of rockets in corresponding routes. i.e.:
```javascript
// get rockets data from the store
const rockets = useSelector(state => state.rockets);
```

- You can style the whole application "by hand" or you could use [React Bootstrap](https://react-bootstrap.github.io/), a UI library that could speed up the process. This is a popular library and working with its components would be good practice.
- Render a list of rockets (as per design). For the image of a rocket use the first image in the array of `flickr_images`.

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
- Use useSelector(), Redux Hook, to select the state slices and render lists of rockets in corresponding routes.
- Render a list of rockets as per design

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+import { fetchRockets } from './Rockets/rockets';
 import './App.css';
 
 function App() {
+  const rockets = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, []);
```
Comment (qwibbler) [inline src/App.js]: I think it would be better to put the rockets logic in the rockets page.

**Review by qwibbler — CHANGES_REQUESTED** [review]
The project is very well done, I just have one suggested change below.

File: src/App.js (hunk 1 above)
Comment (Lucas-Masaba) [reply src/App.js]: Thank you for your suggestion. I just made the required changes

**Review by qwibbler — APPROVED** [review]
Great!

**Pull request merged**